cli/cmd/inject.go

```diff
@@ -212,12 +214,53 @@ func injectPodSpec(t *v1.PodSpec, controlPlaneDNSNameOverride string, options *i
 			{Name: "CONDUIT_PROXY_PRIVATE_LISTENER", Value: fmt.Sprintf("tcp://127.0.0.1:%d", options.outboundPort)},
 			{Name: "CONDUIT_PROXY_PUBLIC_LISTENER", Value: fmt.Sprintf("tcp://0.0.0.0:%d", options.inboundPort)},
 			{
-				Name:      "CONDUIT_PROXY_POD_NAMESPACE",
+				Name:      PodNamespaceEnvVarName,
 				ValueFrom: &v1.EnvVarSource{FieldRef: &v1.ObjectFieldSelector{FieldPath: "metadata.namespace"}},
 			},
 		},
 	}
 
+	if options.enableTLS() {
+		yes := true
+
+		configMapVolume := v1.Volume{
+			Name: "conduit-trust-anchors",
+			VolumeSource: v1.VolumeSource{ConfigMap: &v1.ConfigMapVolumeSource{
+				LocalObjectReference: v1.LocalObjectReference{Name: k8s.TLSTrustAnchorConfigMapName},
+				Optional:             &yes,
+			}},
+		}
+		secretVolume := v1.Volume{Name: "conduit-secrets", VolumeSource: v1.VolumeSource{
+			Secret: &v1.SecretVolumeSource{
+				SecretName: identity.ToSecretName(),
+				Optional:   &yes,
+			},
+		}}
+
+		base := "/var/conduit-io"
+		configMapBase := base + "/trust-anchors"
+		secretBase := base + "/identity"
+		tlsEnvVars := []v1.EnvVar{
+			{Name: "CONDUIT_PROXY_TLS_TRUST_ANCHORS", Value: configMapBase + "/" + k8s.TLSTrustAnchorFileName},
+			{Name: "CONDUIT_PROXY_TLS_CERT", Value: secretBase + "/" + k8s.TLSCertFileName},
+			{Name: "CONDUIT_PROXY_TLS_PRIVATE_KEY", Value: secretBase + "/" + k8s.TLSPrivateKeyFileName},
+			{
+				Name: "CONDUIT_PROXY_TLS_POD_IDENTITY",
+				Value: identity.ToDNSName(),
+			},
+			{Name: "CONDUIT_PROXY_CONTROLLER_NAMESPACE", Value: controlPlaneNamespace}	,
```

Comment:
Can you run `goimports` on this file? It should remove the tab on this line between `}` and `,`.